Makefile.am

```diff
@@ -58,7 +58,7 @@ all-local: .stamp-distutils-in-builddir config.py cupshelpers/config.py
 
 # Use distutils to install the module.
 install-exec-local: .stamp-distutils-in-builddir
-	$(PYTHON) setup.py install --install-layout=deb --root=$(DESTDIR)
+	$(PYTHON) setup.py install --install-layout=deb --root=$(DESTDIR) --prefix=$(prefix)
```

Comment:
Looks like I was wrong about this. In theory it's the right thing to do, but actually distcheck doesn't set DESTDIR and distutils takes an empty --root= argument to mean `.`.